dashboards/GatherMetrics.json

```diff
@@ -24,10 +24,104 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
-  "id": 5,
+  "id": 22,
   "links": [],
   "liveNow": false,
   "panels": [
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "PBFA97CFB590B2093"
```

Comment:
this might not work. As I always mention lets use 

"datasource": "Prometheus",

you already have different uuid in other panels. I would change all fields to what I mentioned or use that uid which is already in the doc

"datasource": {
        "type": "prometheus",
        "uid": "000000001"
      },